main/res/values/strings.xml

```diff
@@ -248,6 +248,7 @@
     <string name="warn_search_help_keyword">Enter all or part of a geocache\'s name. For example \"Night Cache\".</string>
     <string name="warn_search_help_user">Enter a username from Geocaching.com.</string>
     <string name="warn_search_help_tb">Enter the code for a trackable. For example \"TB29QMZ\".</string>
+    <string name="warn_search_no_active_connectors">No active connectors found for this search</string>
```

Comment:
There should already be a comparable string available. It is e.g. shown when you search by keyword while no connector is active.

Reply:
you are right, we could use this one (haven't seen):
`<string name="warn_no_connector">No currently activated geocaching services provide this type of search.</string>`

or we could change the string (at least for geocode search) to something like "cannot search, no connector activated" which points the user more prominently to the root cause

Reply:
IMHO for now the existing should be fine.

Reply:
While this issue has already been resolved, please be aware that "connector" is an implementation internal term and should _never_ appear in user messages. When writing new resource strings, they must always fit to the mental model of the user, not to the implementation.